src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -28,27 +28,30 @@
     private final Logger logger = LogsCenter.getLogger(getClass());
 
     private Stage primaryStage;
+    private ResultDisplay resultDisplay;
+    private TagListPanel tagListPanel;
+    private LastInputDisplay lastInputDisplay;
```

Comment:
Are these 3 variables no longer independent Ui parts?

`// Independent Ui parts residing in this Ui container`

Reply:
I think I put them in a wrong place while modifying, did not notice that

Reply:
I updated the PR to move them under the "independent Ui part" comment. Thank you for pointing that out.